src/main/java/frc/robot/subsystems/drive/CommandSwerveDrivetrain.java

```diff
@@ -238,11 +257,16 @@ private void controlDrivetrain() {
     }
 
     private Rotation2d calculateDesiredHeading(Pose2d current, Pose2d target) {
-        double robotXAnticipated =
-                current.getX() + (getRobotVelocity.get().getX() * DriveConstants.anticipationTime);
+        Translation2d robotVelocityAdjusted =
+                getRobotVelocity.get().times(DriveConstants.anticipationTime);
+
+        if (Math.abs(robotVelocityAdjusted.getNorm())
+                < DriveConstants.minimumAnticipationVelocity) {
```

Comment:
This constant is 0, so are we doing anything here? Or is this just setup for the future.

Either way feel free to leave as-is, I'm just curious about what is going on here

Reply:
It is setup for the future if needed. I thought that I'd have to use it but then made it work without.